<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowTimeCourse</class>
 <widget class="QWidget" name="WindowTimeCourse">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>309</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <widget class="WidgetTimeCoursePlot" name="widgetPlot" native="true">
        <property name="minimumSize">
         <size>
          <width>350</width>
          <height>250</height>
         </size>
        </property>
        <property name="focusPolicy">
         <enum>Qt::StrongFocus</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>5</width>
         <height>5</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Scale</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditScale"/>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxAutoScale">
       <property name="text">
        <string>Auto</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxMaxScale">
       <property name="text">
        <string>Max</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WidgetTimeCoursePlot</class>
   <extends>QWidget</extends>
   <header>WidgetTimeCoursePlot.h</header>
   <container>1</container>
   <slots>
    <slot>SetAutoScale(bool)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>checkBoxAutoScale</sender>
   <signal>toggled(bool)</signal>
   <receiver>WindowTimeCourse</receiver>
   <slot>OnCheckAutoScale(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>291</x>
     <y>289</y>
    </hint>
    <hint type="destinationlabel">
     <x>283</x>
     <y>398</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxMaxScale</sender>
   <signal>toggled(bool)</signal>
   <receiver>WindowTimeCourse</receiver>
   <slot>OnCheckMaxScale(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>358</x>
     <y>287</y>
    </hint>
    <hint type="destinationlabel">
     <x>353</x>
     <y>493</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditScale</sender>
   <signal>returnPressed()</signal>
   <receiver>WindowTimeCourse</receiver>
   <slot>OnLineEditScaleReturnPressed()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>273</x>
     <y>288</y>
    </hint>
    <hint type="destinationlabel">
     <x>270</x>
     <y>388</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnLineEditScaleReturnPressed()</slot>
  <slot>OnCheckAutoScale(bool)</slot>
  <slot>OnCheckMaxScale(bool)</slot>
 </slots>
</ui>
